### PR TITLE
Fix normal shutdown to ensure any message processing are given time to finish and ack

### PR DESCRIPTION
**Pull request by bbalser**

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -12,7 +12,7 @@ defmodule Elsa.MixProject do
       deps: deps(),
       elixirc_paths: elixirc_paths(Mix.env()),
       test_paths: test_paths(Mix.env()),
-      dialyzer: [flags: [:no_fail_call, :no_return, :no_fun_app], plt_file: {:no_warn, ".plt/dialyzer.plt"}]
```
Comment (jeffgrunewald) [inline mix.exs]: Don’t you give Johnson the satisfaction!

File: test/unit/test_helper.exs [hunk 2]
```diff
@@ -1,4 +1,4 @@
-ExUnit.start()
+ExUnit.start(excude: [:skip])
```
Comment (jeffgrunewald) [inline test/unit/test_helper.exs]: Misspelled “exclude”

File: lib/elsa/group/manager.ex [hunk 3]
```diff
@@ -196,8 +208,11 @@ defmodule Elsa.Group.Manager do
   end
 
   def terminate(reason, state) do
-    Logger.info("#{__MODULE__} : Terminating #{state.connection}")
-    WorkerManager.stop_all_workers(state.workers)
+    Logger.debug(fn -> "#{__MODULE__} : Terminating #{state.connection}" end)
```
Comment (bbrewer-papa-x) [inline lib/elsa/group/manager.ex]: Just makes sense

File: test/integration/elsa/shutdown_test.exs [hunk 4]
```diff
@@ -0,0 +1,80 @@
+defmodule Elsa.ShutdownTest do
+  use ExUnit.Case
+  use Divo
+  import AssertAsync
+
+  @endpoints [localhost: 9092]
+
+  test "non direct ACKs duplicate data" do
```
Comment (bbrewer-papa-x) [inline test/integration/elsa/shutdown_test.exs]: So happy my test is slightly less garbage now

File: test/support/assert_async.ex [hunk 5]
```diff
@@ -0,0 +1,42 @@
+defmodule AssertAsync do
```
Comment (bbrewer-papa-x) [inline test/support/assert_async.ex]: There is also a variant of this in the `assertions` package on hex, if you didn't try it yet and hate it

**Pull request merged**